src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
-export interface Figure {
-
+function roundTo100(digits: number): number {
+  return Math.floor(digits * 100) / 100;
 }
 
-export class Triangle {
+type Shape = 'triangle'| 'circle'| 'rectangle';
+type Color = 'red'| 'green'| 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
+
+  public color;
+
+  public a;
+
+  public b;
+
+  public c;
+
+  constructor(color: Color, a: number, b: number, c: number) {
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
```

Comment:
don't repeat yourself, you can add variables to the constructor once (constructor(public color...){} )

Reply:
thanks for review)